recipes/gsshapyorm/meta.yaml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "gsshapyorm" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "2cf483adbdacc100ffbb3c449279c4824e1d0bae6d0f600696cf773c4a56654c" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  skip: True  # [py2k]
```

Comment:
I think this recipe can be ``noarch`` and you can limit the python version on the ``requirements``